main/guides/orchestration/getting-started/key-concepts.md

```diff
@@ -1,87 +1,67 @@
-
-  // deposit funds from user seat to LocalChainAccount
-...
-  const transferMsg = orcUtils.makeOsmosisSwap({ ... });
-
-  try {
-    await localAccount.transferSteps(give.Stable, transferMsg);
-    await omniAccount.delegate(offerArgs.validator, offerArgs.staked);
-  } catch (e) {
-    console.error(e);
-  }
-};
+const brandInfo = orchestrator.getBrandInfo('denom')
 ```
 
-### ChainHub
-
-The `makeChainHub` utility manages the connections and metadata for various blockchain networks. It creates a new `ChainHub` instance implementing the [`ChainHubI`](https://github.com/Agoric/agoric-sdk/blob/000693442f821c1fcea007a2df740733b1f75ebe/packages/orchestration/src/exos/chain-hub.js#L70-L80C4) interface.
-
-It simplifies accessing and interacting with multiple chains, providing a unified interface for the orchestration logic to manage cross-chain operations effectively.
-ChainHub also allows dynamic registration and use of chain and connection information.
+- `asAmount` converts a denom amount to an `Amount` with a brand. See [asAmount](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.Orchestrator#asAmount).
 
 ```javascript
-const chainHub = makeChainHub(remotePowers.agoricNames)
+const amount = orchestrator.asAmount({ denom: 'uatom', value: 1000n })
+```
 
-// Register a new chain with its information
-chainHub.registerChain(chainKey, chainInfo)
+## Orchestration Account
 
-// Register a connection between the Agoric chain and the new chain
-chainHub.registerConnection(
-  agoricChainInfo.chainId,
-  chainInfo.chainId,
-  connectionInfo
-)
-```
+Orchestration accounts are a key concept in the Agoric Orchestration API, represented by the [`OrchestrationAccountI`](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.OrchestrationAccountI)
+interface. These accounts provide high-level operations for managing accounts on remote chains, allowing seamless
+interaction and management of interchain accounts. The orchestration accounts abstract the complexity of interchain
+interactions, providing a unified and simplified interface for developers.
 
-In this example, `chainHub` is used to register a new chain and establish a connection between the Agoric chain and the newly registered chain.
+### Account Creation
 
-### Orchestration Account
+- `makeAccount` (for a chain object) creates a new account on local and/or remote chain as below.
 
-Orchestration accounts are a key concept in the Agoric Orchestration API, represented by the [`OrchestrationAccountI`](https://agoric-sdk.pages.dev/interfaces/_agoric_orchestration.OrchestrationAccountI) interface. These accounts provide high-level operations for managing accounts on remote chains, allowing seamless interaction and management of interchain accounts. The orchestration accounts abstract the complexity of interchain interactions, providing a unified and simplified interface for developers.
+```javascript
+const [agoric, remoteChain] = await Promise.all([
```

Comment:
not sure its helpful, but I find myself being asked, "what does promise.all do?" more often that I expected when showing this 👀